app/Http/Controllers/MessagesController.php

```diff
@@ -92,7 +92,9 @@ public function sendMessage(Message $message)
     {
         // Get competition with activity.
         $competitionId = request('competition_id');
-        // $competition = null;
+
+        $contest = Contest::find(request('contest_id'));
+        $this->manager->appendCampaign($contest);
```

Comment:
Shit. I think it's done incorrectly there too @_@ I don't know how that is working correctly...